ffi/capi/config.toml

```diff
@@ -11,3 +11,8 @@ module-name = "icu"
 # lib-name becomes a namespace, we don't want to clash
 # with ICU4C
 lib-name = "icu4x"
+
+[kotlin]
```

Comment:
Does it need to be added to GitHub actions or does it get picked up automatically?

Reply:
It's a part of the gen-kotlin task

We don't have any tests yet. I need to get that working.